lang/de_DE.yml

```diff
@@ -1,11 +1,11 @@
 de_DE:
-  EditableFormField:
+  SilverStripe\UserForms\Model\EditableFormField:
     php:
       SilverStripe\UserForms\Model\EditableFormFieldTYPE_EMPTY: 'Wähle Feld-Typ'
```

Comment:
That's an invalid key. Run a `tx pull -a --force` to get the latest translations. I think this might have already been fixed in the `5` branch because I got a valid translation for `SilverStripe\UserForms\Model\EditableFormField.TYPE_EMPTY`

Reply:
Yeah, it's an invalid key because, it's a CMS3 key?  The point of this PR is to update it to a CMS4 key so that the translations actually get used

Judging by the other pull-request you oppened/closed, running `tx pull -a --force` will incorrectly delete these translations from the .yml file https://github.com/silverstripe/silverstripe-userforms/pull/1077/files#r642144981

I've updated these keys to remove the weird `php:` from the yml and made them look like regular translation definitions.

Note these ARE used in php e.g. `_t(__CLASS__.'.TYPE_EMPTY' ... `

I think the correct solution here is:
1) merge this
2) run `tx push -s` then `tx push -t` - this will push the source file update (en.yml) then all the translation files aftewards